### PR TITLE
refactor: Customer queries

### PR DESCRIPTION
**Pull request by brianl9995**
Signed-off-by: Brian

### CODE REVIEW

File: run-payinv.sh [hunk 1]
```diff
@@ -3,6 +3,6 @@
 python src/manage.py migrate
 python src/manage.py check --deploy
 python src/manage.py collectstatic --no-input
-python src/manage.py compilemessages
+python src/manage.py compilemessages -l es
```
Comment (roramirez) [inline run-payinv.sh]: Well, actually only have the messages for `es_ES`. You  should change by this key instead only `es`

File: src/customers/models.py [hunk 2]
```diff
@@ -37,16 +37,14 @@ def total_sales(self):
...
-            sale_id__in=sales_id).aggregate(
+            sale__in=self.sale_set.all()).aggregate(
             Sum('total_value'))['total_value__sum'] or 0
 
     @property
     def total_invoices(self):
-        sales_id = self.sale_set.all().values('id')
         return invoices.models.Invoice.objects.filter(
-            sale_id__in=sales_id).aggregate(
+            sale__in=self.sale_set.all()).aggregate(
```
Comment (roramirez) [inline src/customers/models.py]: Greats!

File: run-payinv.sh (hunk 1 above)
Comment (brianl9995) [reply run-payinv.sh]: resolved in 7fc2160c2c9bbb9568a659ff3edf2526142d33fc

Comment (roramirez) [reply run-payinv.sh]: Good!

**Comment by roramirez** [comment]
Thanks @brianl9995

**Pull request merged**